src/ripple/app/ledger/impl/LedgerToJson.cpp

```diff
@@ -159,7 +167,10 @@ fillJsonTx(
         txJson[jss::validated] = validated;
         if (validated)
         {
-            txJson[jss::ledger_index] = to_string(fill.ledger.seq());
+            auto const seq = fill.ledger.seq();
+            txJson[jss::ledger_index] = (fill.context->apiVersion > 1)
```

Comment:
This PR is meant to make the change trivial switching between API version 3 and version 2, which is why it looks like this. If this is indeed released for API version 2 then I have another PR lined up to remove redundant conditional expression here and instead unconditionally set `ledger_index` to a number (since this is inside `else if (fill.context->apiVersion > 1)` block starting at line 140)